test/packages/caver.klay.accounts.js

```diff
@@ -64,8 +64,8 @@ function checkHashMessage(hashed, originMessage) {
 }
 
 function isKeystoreV3(data, { address }) {
```

Comment:
It seems that the function name should be changed to `isKeyStoreV4`.

Reply:
Oh, right ! I will change the name.